app/code/Magento/Quote/Model/Quote/Validator/MinimumOrderAmount/ValidationMessage.php

```diff
@@ -18,23 +18,22 @@ class ValidationMessage
     private $scopeConfig;
 
     /**
-     * @var \Magento\Framework\Locale\CurrencyInterface
+     * @var \Magento\Framework\Pricing\Helper\Data
      */
-    private $currency;
-
+    private $priceHelper;
     /**
      * @param \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
      * @param \Magento\Store\Model\StoreManagerInterface $storeManager
-     * @param \Magento\Framework\Locale\CurrencyInterface $currency
+     * @param \Magento\Framework\Pricing\Helper\Data $priceHelper
      */
     public function __construct(
         \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig,
         \Magento\Store\Model\StoreManagerInterface $storeManager,
-        \Magento\Framework\Locale\CurrencyInterface $currency
+        \Magento\Framework\Pricing\Helper\Data $priceHelper
```

Comment:
please don't remove parameters from constructor and
add dependency in a backward compatible manner according to http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/#adding-a-constructor-parameter